modules/ref-dashboard-configuration-options.adoc

```diff
@@ -68,41 +83,61 @@ On the *Deploy model* dialog when using the single-model serving platform:
-| `spec.dashboardConfig.disableNIMModelServing` | `false` | Enables the ability to select NVIDIA NIM as a model-serving platform. To disable this ability, set the value to `true`.
-| `spec.dashboardConfig.disablePerformanceMetrics` | `false` | Shows the *Endpoint Performance* tab on the *Model deployments* page. To hide this tab, set the value to `true`.
-| `spec.dashboardConfig.disablePipelines` | `false` | Shows the *Data science pipelines* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
-| `spec.dashboardConfig.disableProjects` | `false` | Shows the *Data science projects* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
-| `spec.dashboardConfig.disableProjectSharing` | `false` | Allows users to share access to their data science projects with other users. To prevent users from sharing data science projects, set the value to `true`.
-| `spec.dashboardConfig.disableServingRuntimeParams` | `false` | Shows the *Configuration parameters* section in the *Deploy model* dialog and the *Edit model* dialog when using the single-model serving platform. To hide this section, set the value to `true`.
-| `spec.dashboardConfig.disableStorageClasses` | `false` | Shows the *Settings → Storage classes* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
-| `spec.dashboardConfig.disableSupport` | `false` | Shows the *Support* menu item when a user clicks the Help icon in the dashboard toolbar. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disableModelServing` | `false` | Shows the *Models → Model deployments* menu item in the dashboard navigation menu, and the *Models* tab in data science projects. To hide these items, set the value to `true`. 
+| `spec.dashboardConfig.` +
+`disableNIMModelServing` | `false` | Enables the ability to select NVIDIA NIM as a model-serving platform. To disable this ability, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disablePerformanceMetrics` | `false` | Shows the *Endpoint Performance* tab on the *Model deployments* page. To hide this tab, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disablePipelines` | `false` | Shows the *Data science pipelines* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disableProjects` | `false` | Shows the *Data science projects* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.` +
+`disableProjectScoped` | `false` | Distinguishes between global items and project-scoped items (if project-scoped items exist) in the {productname-short} web console. This option applies to workbenches, hardware profiles, accelerator profiles, and model-serving runtimes for KServe. To disable this functionality, set the value to `true`.
```

Comment:
One small change: "This applies to workbench images, hardware profiles, accelerator profiles and ...". Also hardware profiles and accelerator profiles exist in both workbenches and model deployments (Kserve). Workbench images exist in workbenches and model serving runtimes in Kserve. Is there a need to specify this? Or is this obvious?

Reply:
Then I think it should be "workbench images" instead of "workbenches"

Reply:
@manaswinidas My apologies, I missed that comment earlier. Updated now, and thank you!